.github/workflows/app-dh-healthchecks.yml

```diff
@@ -42,3 +42,20 @@ jobs:
 
       - name: Run healthchecks
         run: npx nx run e2e-dh:e2e:health
+
+  #
+  # Send notification to teams channel if healthchecks failed
+  #
+  dispatch_failed:
+    needs:
+      [
+        healthchecks
+      ]
+    if: |
+      always() &&
+      contains(needs.*.result, 'failure')
+    uses: Energinet-DataHub/.github/.github/workflows/notify-team.yml@v10
+    with:
+      team_name: TexasRangers
```

Comment:
Should this not be a different team name?

Reply:
It's a "legacy" name from the time we were part of The Outlaws. For now it needs to be TexasRangers. See 
https://github.com/Energinet-DataHub/.github/blob/main/.github/workflows/notify-team.yml#L56C51-L57

Reply:
What @dzhavat said :)